packages/suite/src/utils/suite/oauth.ts

```diff
@@ -0,0 +1,73 @@
+import { getPrefixedURL } from '@suite-utils/router';
+import { METADATA } from '@suite-actions/constants';
+import { Deferred, createDeferred } from '@suite-utils/deferred';
+import { urlHashParams } from '@suite-utils/metadata';
+/**
+ * For desktop, always use oauth_receiver.html from trezor.io
+ * For web, use oauth_receiver.html hosted on the same origin (localhost/sldev/trezor.io)
+ */
+export const getOauthReceiverUrl = () => {
+    // @ts-ignore
+    if (!window.ipcRenderer) {
+        return `${window.location.origin}${getPrefixedURL('/static/oauth/oauth_receiver.html')}`;
+    }
+
+    // todo: upload oauth_receiver.html from suite to trezor.io (it contains code to handle ipcRenderer);
+    // return 'https://beta-wallet.trezor.io/oauth_receiver.html';
+    // todo: temporarily herokuapp is used.
+    return 'https://track-suite.herokuapp.com/oauth/';
```

Comment:
dont forget!

Reply:
issue for that https://github.com/trezor/trezor-suite/issues/2244